packages/dita-example-sx-modules-xsd-common-element-mod/src/configureSxModule.js

```diff
@@ -486,7 +487,7 @@ export default function configureSxModule(sxModule) {
 
 	Object.keys(NOTE_VISUALIZATION_BY_TYPE).forEach(function(noteType) {
 		const noteVisualization = NOTE_VISUALIZATION_BY_TYPE[noteType];
-		configureProperties(sxModule, 'self::note[@type="' + noteType + '"]', {
+		configureProperties(sxModule, xq`self::note[@type="${noteType}"]`, {
```

Comment:
The extra quotes should not be needed: the string interpolations should kick in to make that noteType a variable instead